src/Tools/ExternalAccess/Razor/RazorGlobalOptions.cs

```diff
@@ -29,6 +29,18 @@ public RazorGlobalOptions(IGlobalOptionService globalOptions)
         public RazorAutoFormattingOptions GetAutoFormattingOptions()
             => new(_globalOptions.GetAutoFormattingOptions(LanguageNames.CSharp));
 
+        public bool UseTabs
+        {
+            get => _globalOptions.GetOption(RazorLineFormattingOptionsStorage.UseTabs);
+            set => _globalOptions.SetGlobalOption(new OptionKey(RazorLineFormattingOptionsStorage.UseTabs), value);
+        }
+
+        public int TabSize
+        {
+            get => _globalOptions.GetOption(RazorLineFormattingOptionsStorage.TabSize);
+            set => _globalOptions.SetGlobalOption(new OptionKey(RazorLineFormattingOptionsStorage.TabSize), value);
+        }
+
 #pragma warning disable IDE0060 // Remove unused parameter
```

Comment:
😬 Everything below this line should probably be slated for removal

Reply:
Yeah, would need to rewrite Razor test infra :(